<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicineView</class>
 <widget class="QWidget" name="MedicineView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>药品信息管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelMedicineName">
       <property name="text">
        <string>药品名称：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputMedicineName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDosage">
       <property name="text">
        <string>剂量：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputDosage"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelStock">
       <property name="text">
        <string>库存数量：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inputStock"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>添加药品</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>删除药品</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="btnQuery">
       <property name="text">
        <string>查询药品</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>导出药品信息</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidgetMedicines">
       <property name="columnCount">
        <number>3</number>
       </property>
       <property name="horizontalHeaderItem">
        <string>药品名称</string>
       </property>
       <property name="horizontalHeaderItem">
        <string>剂量</string>
       </property>
       <property name="horizontalHeaderItem">
        <string>库存数量</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
